users/users.go

```diff
@@ -80,11 +81,21 @@ func isEmail(email string) bool {
 	}
 
 	hs := strings.Split(host, dotSeparator)
-	if len(hs) != 2 {
+	if len(hs) < 2 {
 		return false
 	}
 	domain, ext := hs[0], hs[1]
 
+	// Check subdomain and validate
+	if len(hs) == 3 {
```

Comment:
Why 3? Can there be multiple subdomains (I do not know, but theorethically stere can be sub-subdomain)?